src/pdeathsig-helper.c

```diff
@@ -0,0 +1,16 @@
+// Adapted from: https://stackoverflow.com/a/13280892
+// Super tiny helper to auto-exit command when parent exits
+#include <unistd.h>
+#include <sys/prctl.h>
+#include <signal.h>
+#include <err.h>
+
+int main(int argc, char **argv) {
+    if (argc < 2)
+        errx(1, "Usage: %s COMMAND [ARGUMENTS...]", argv[0]);
+    if (prctl(PR_SET_PDEATHSIG, SIGTERM, 0, 0, 0) < 0)
```

Comment:
I think this is just `setpriv --pdeathsig SIGTERM -- ` right?

Reply:
Yup, much simpler!